mteb/models/bge_models.py

```diff
@@ -56,3 +56,12 @@ def encode_corpus(
     revision="a5beb1e3e68b9ab74eb54cfd186867f64f240e1a",
     release_date="2023-09-11",  # initial commit of hf model.
 )
+
+bge_large_en_v1_5 = ModelMeta(
+    loader=partial(BGEWrapper, model_name="BAAI/bge-large-en-v1.5"),  # type: ignore
+    name="BAAI/bge-large-en-v1.5",
+    languages=["eng_Latn"],
+    open_source=True,
+    revision="d4aa6901d3a41ba39fb536a557fa166f842b0e09",
+    release_date="2023-09-12",  # initial commit of hf model.
+)
```

Comment:
Don't we need `normalize_embeddings=True` when calling `self.mdl.encode`? 🤔
See
(1) https://huggingface.co/BAAI/bge-large-en-v1.5#using-sentence-transformers
(2) https://github.com/UKPLab/sentence-transformers/blob/2dee8c22f618df3f34f55d406bf97aa88383728a/sentence_transformers/SentenceTransformer.py#L430

Maybe worth checking on a quick retrieval dataset like SciFact if we get similar scores to what the model gets on the mteb leaderboard

Reply:
Good catch! I've run `mteb run -t SciFact -m BAAI/bge-large-en-v1.5 --model_revision d4aa6901d3a41ba39fb536a557fa166f842b0e09`. 

`bge-large-en-v1.5` gives 74.63 from mteb (this PR) and 74.61 from the leaderboard.

Reply:
Rerunning with `normalize_embeddings=True` yields 74.63 as well.